src/TeachABit.Front/src/components/navigation/Navigation.tsx

```diff
@@ -45,35 +46,57 @@ export default function Navigation() {
                         flexDirection: "column",
                         height: "100%",
                         width: "100%",
+                        backgroundColor: "#D9D9D9",
                     }}
                 >
                     <List sx={{ width: "100%" }}>
-                        <ListItem disablePadding>
+                        <ListItem>
+                            <ListItemIcon sx={{
+                                display: 'flex',
+
+
+                            }}>
+
+                                    <img src={Logo} alt="Teach A Bit Logo"
+                                         style={{width: "15%", height: "15%", marginBottom:40}}/>
+
+                            </ListItemIcon>
+                        </ListItem>
+                        <ListItem  disablePadding>
                             <ListItemButton
                                 onClick={() => navigate("/tecajevi")}
                             >
                                 <ListItemIcon>
-                                    <BookIcon />
+                                    <BookIcon  fontSize="large" style={{ color: "922728" }} />
                                 </ListItemIcon>
-                                <ListItemText primary="Tečajevi" />
```

Comment:
Ovdje bi također trebalo koristiti klasu, to olakšava kasnije izmjene za sve ListItemText komponente koje imaju tu klasu. Općenito bi trebalo koristiti inline css samo ako se ne planira nigdje drugdje koristiti za isto.